### DISCUSSION TITLE
latexindent stopped working

### DISCUSSION THREAD

**Discussion by RobertHarper** [James-Yu/LaTeX-Workshop#3606] [Q&A]
Hello, Please excuse my naivete, but I wonder whether anyone knows of an issue with latexindent whereby it errors out suggesting that I lack module HomeDir?  I surmise that it has something to do with Perl, but anyway it was working, then an update caused it to not work any more.

For reference, I am on Mac OS, using MacTeX from the TeXLive distribution.   It would appear that an update to the TeX distro caused this error, which I see arise while using this extension in Visual Studio.

Thank you.

**Comment by jlelong** [comment]
This is not related to this extension but to some missing perl modules on your system. Install the perl module `File:HomeDir`. You may need to install `Mac:SystemDirectory` first.
